framework/app.mk

```diff
@@ -257,11 +256,18 @@ else
   app_plugin_deps :=
 endif
 
-app_LIBS       := $(app_LIB) $(app_LIBS)
-app_LIBS_other := $(filter-out $(app_LIB),$(app_LIBS))
```

Comment:
so this one was unused?

Reply:
As far as I (and CIVET) could tell at least.